### PR TITLE
[14.0][FW][IMP/FIX] rma, rma_sale: forward ports

### PR DESCRIPTION
**Pull request by chienandalu**
Forwards:

- https://github.com/OCA/rma/pull/252
- https://github.com/OCA/rma/pull/261
- https://github.com/OCA/rma/pull/262
- https://github.com/OCA/rma/pull/264
- https://github.com/OCA/rma/pull/282

cc @Tecnativa TT33572 TT34660 TT35322

please review @victoralmau @ernestotejeda @pedrobaeza

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
and let's put it a higher bump:

/ocabot merge major

File: rma/i18n/es.po [hunk 1]
```diff
@@ -10,43 +10,52 @@ msgstr ""
 "PO-Revision-Date: 2022-01-17 12:39+0000\n"
 "Last-Translator: xavigutipujol <jgutierrez@pyming.com>\n"
 "Language-Team: \n"
-"Language: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+<<<<<<< HEAD
```
Comment (pedrobaeza) [inline rma/i18n/es.po]: These conflicts must be solved

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: rma/i18n/es.po (hunk 1 above)
Comment (chienandalu) [reply rma/i18n/es.po]: This one escaped from my local pre-commit run... Fixed

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**